<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrepareReagentDialog</class>
 <widget class="QDialog" name="PrepareReagentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>988</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>膜式</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <property name="dialogStyle" stdset="0">
   <string>BGStyle</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widgetTop" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>20</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>20</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>试剂准备</string>
              </property>
              <property name="lblStyle" stdset="0">
               <string>lblNormal</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonClose">
              <property name="minimumSize">
               <size>
                <width>26</width>
                <height>26</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>26</width>
                <height>26</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QWidget" name="vLayoutWidget" native="true">
       <property name="widgetStyle" stdset="0">
        <string>BGStyle</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>30</number>
        </property>
        <property name="topMargin">
         <number>40</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>分配或延迟，每个已用试剂</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnSmall">
            <property name="text">
             <string>小</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnBig">
            <property name="text">
             <string>大</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>1</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnFlash">
            <property name="minimumSize">
             <size>
              <width>128</width>
              <height>128</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="btnStyle" stdset="0">
             <string>btnReagentStyle</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonSmall">
            <property name="minimumSize">
             <size>
              <width>128</width>
              <height>128</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="btnStyle" stdset="0">
             <string>btnReagentStyle</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnReady">
            <property name="minimumSize">
             <size>
              <width>128</width>
              <height>128</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="btnStyle" stdset="0">
             <string>btnReagentStyle</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnDelay">
            <property name="minimumSize">
             <size>
              <width>128</width>
              <height>128</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="btnStyle" stdset="0">
             <string>btnReagentStyle</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="hLayPump">
            <property name="spacing">
             <number>2</number>
            </property>
            <property name="leftMargin">
             <number>10</number>
            </property>
            <property name="topMargin">
             <number>10</number>
            </property>
            <property name="rightMargin">
             <number>10</number>
            </property>
            <property name="bottomMargin">
             <number>10</number>
            </property>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QPushButton" name="btnConfig">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>确  定</string>
         </property>
         <property name="btnStyle" stdset="0">
          <string>normalBtn</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>btnConfig</tabstop>
  <tabstop>pushButtonClose</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
